packages/x-components/src/x-modules/queries-preview/events.types.ts

```diff
@@ -17,4 +19,16 @@ export interface QueriesPreviewXEvents {
    * Payload: The query whose preview has been removed.
    */
   QueryPreviewUnmountedHook: string;
+  /**
+   * User has clicked on a query preview.
+   * Payload: The {@link QueryPreviewInfo | query preview info}
+   * of the selected {@link QueryPreviewItem | item}.
+   */
+  UserAcceptedAQueryPreview: QueryPreviewInfo;
+  /**
+   * The selected query preview has changed.
+   * Payload: The new selected query preview with extra params or the initial module
+   * params if the query has been cleared.
+   */
```

Comment:
Hi!
If I am not wrong, I think this doc should be updated to be more precise, at least the payload description, as now it is not a change event that will always be fired.